README.md

```diff
@@ -1,2 +1,254 @@
+    <xs:complexType>
+      <xs:sequence>
+        <xs:element maxOccurs="unbounded" minOccurs="1" ref="interpreter"/>
+      </xs:sequence>
+    </xs:complexType>
+  </xs:element>
+  <xs:element name="interpreter">
+    <xs:complexType>
+      <xs:sequence>
+        <xs:element maxOccurs="unbounded" minOccurs="1" ref="configuration"/>
+      </xs:sequence>
+      <xs:attribute name="id" use="required" type="xs:ID"/>
+    </xs:complexType>
+  </xs:element>
+  <xs:element name="configuration">
+    <xs:complexType>
+      <xs:attribute name="analyser" use="required"/>
+      <xs:attribute name="parser" use="required"/>
+      <xs:attribute name="type" use="required"/>
+    </xs:complexType>
+  </xs:element>
+</xs:schema>
+```
+
+**monitoring-config.xml**
+```
+<?xml version="1.0" encoding="utf-8"?>
+<config xmlns="http://www.atb-bremen.de"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.atb-bremen.de monitoring-config.xsd">
+
+    <indexes>
+        <index id="index-git" location="target/indexes/git"/>
+    </indexes>
+
+    <datasources>
+        <datasource id="datasource-git" type="messagebroker"
+                    monitor="de.atb.context.monitoring.monitors.GitMonitor"
+                    uri=""
+                    options="server=localhost&amp;port=5672&amp;exchange=smartclide-monitoring&amp;topic=monitoring.git.*&amp;dle-topic=dle.git.commits"
+                    class="de.atb.context.monitoring.config.models.datasources.MessageBrokerDataSource"/>
+    </datasources>
+
+    <interpreters>
+        <interpreter id="interpreter-git">
+            <configuration type="*"
+                           parser="de.atb.context.monitoring.parser.GitParser"
+                           analyser="de.atb.context.monitoring.analyser.GitAnalyser"/>
+        </interpreter>
+    </interpreters>
+
+    <monitors>
+        <monitor id="monitor-git" datasource="datasource-git" interpreter="interpreter-git" index="index-git"/>
+    </monitors>
+</config>
+```
+
+The "monitoring-config.xml" configuration file can also be changed by using the Admin UI.
+
+
+### Description
+
+#### indexes
+
+Each index entry has the following mandatory attributes
+
+- id: The unique name of the index
+- location: The URI of the location the index is stored
+
+#### datasources
+
+Each datasource entry has the following mandatory attributes
+
+- id:The unique name of the datasource
+- type:The type of the datasource. Possible values are: file, webservice, database
+- monitor:The class of the monitor to be used. Possible values are:
+    - package de.atb.context.monitoring.monitors.database.DatabaseMonitor
+    - package de.atb.context.monitoring.monitors.file.FileSystemMonitor
+    - package de.atb.context.monitoring.monitors.file.FilePairSystemMonitor
+    - package de.atb.context.monitoring.monitors.file.FileTripletSystemMonitor
+    - package de.atb.context.monitoring.monitors.webservice.WebServiceMonitor
```

Comment:
The monitors / data source implementations added by SmartCLIDE should be listed as well.

Reply:
[x] done